packages/block-editor/src/components/list-view/block.js

```diff
@@ -231,6 +231,10 @@ function ListViewBlock( {
 		selectedClientIds,
 	} );
 
+	// Detect if there is a block in the canvas currently being edited and multi-selection is not happening.
+	const currentlyEditingBlockInCanvas =
+		isSelected && selectedClientIds.lenth === 1;
```

Comment:
There's tiny typo here `lenth` instead of `length`.

```suggestion
		isSelected && selectedClientIds.length === 1;
```